src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -102,14 +119,33 @@ export default {
 		const { loans, totalCount } = await runLoansQuery(
 			this.apollo,
 			{ ...this.flssLoanSearch, ...this.loanSearchState },
-			FLSS_ORIGIN_CATEGORY
+			FLSS_ORIGIN_LENDING_HOME
```

Comment:
Is this component also used on Category pages still? Seems like this should be conditional based on the context.

Reply:
Good catch thanks, I totally forgot it's still on cat pages for some reason I thought we removed it😅😂

Reply:
oh wait - this is actually specific to the loan finding page and not re-used. `QuickFiltersSection` is the parent component on loan finding for `QuickFilters` while it's `LoanChannelCategoryControl` on Category pages. so should be ok!